erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -647,7 +647,7 @@ def get_available_item_locations_for_batched_item(
 		.groupby(sle.warehouse, sle.batch_no, sle.item_code)
 		.having(Sum(sle.actual_qty) > 0)
 		.orderby(IfNull(batch.expiry_date, "2200-01-01"), batch.creation, sle.batch_no, sle.warehouse)
-		.limit(cint(required_qty + total_picked_qty))
+		.limit(get_limit_for_query(required_qty + total_picked_qty))
```

Comment:
Should it always be `math.ceil` :thinking:  ?


---

Also limiting rows here is kinda weird right? We want to keep picking till we hit quantity but since that's not feasible we have to put this weird limit clause (?)

Reply:
Yes we can use math.ceil